packages/babel-generator/src/generators/flow.js

```diff
@@ -270,6 +270,9 @@ export function ObjectTypeAnnotation(node: Object) {
     this.space();
 
     this.printJoin(props, node, {
+      addNewlines(leading) {
```

Comment:
Unrelated to this particular pr, but does the `if (props.length) {` really do anything since it can be `[]`?